.travis.yml

```diff
@@ -114,15 +136,14 @@ jobs:
       install:
         - choco install python --version=$PYVER
         - export PATH="/c/$PYDIR:/c/$PYDIR/Scripts:$PATH"
-        - choco install ffmpeg
         - cmd.exe //c "RefreshEnv.cmd"
         - python -m pip install --upgrade pip
         - python -m pip install --user -r ./.travis/travis-requirements.txt
         - python ./scripts/pycairoinstall.py
         - cmd.exe //c "RefreshEnv.cmd"
         - python -m pip install --user .
       script: 
-        - python -m pytest
+        - python -m pytest --skip_end_to_end -rs
```

Comment:
Same thing for Windows here. We probably want to use a `.bat` script to set up the environment.

Reply:
This comment was supposed to be 3 lines up in `before-install`, but you get the gist.